viper_test.go

```diff
@@ -2281,6 +2281,64 @@ func TestCaseInsensitiveSet(t *testing.T) {
 	}
 }
 
+type overrider struct {
+	Key     string
+	Val     string
+	Enabled bool
```

Comment:
Hmm, the `Enabled` field is used in the `Get(...)` function and in the unit test below.

Reply:
Are you proposing a different way to return that boolean?

Reply:
I see it in the test but in the actual functioning code it does not appear to be used. My thought would be to remove that capability altogether. What use case is it satisfying?

Reply:
So the signature of Get is always whether the key had a value and whether it's enabled or not? I think we should reconsider that interface.

Reply:
Also, the Get signature can't return a string only, this actually brings up a larger point about how Viper works overall. If you have a generic overrider interface it needs to support all the Gets.

Reply:
It looked like even if it returns just a string it works with all of the Gets... I can add some test cases to confirm this.

Reply:
> Actually, on a second look it's in the test only. OK, that's fine except still what's the purpose?

The purpose was to make sure if the second boolean was false, that it would fall back to the other "backup" methods -- whether that's `viper.Set()` or the previous overriders.